lib/ts/types.ts

```diff
@@ -109,4 +116,46 @@ export type RecipeInterface = {
     doesSessionExist: (input: { userContext: any }) => Promise<boolean>;
 
     signOut: (input: { userContext: any }) => Promise<void>;
+
+    getInvalidClaimsFromResponse(input: { response: AxiosResponse | Response }): Promise<ClaimValidationError[]>;
```

Comment:
Missing userContext - I know that it won't be needed as such, but we have to be consistent with other functions.